package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "leaderboard-app",
+  "version": "1.0.0",
+  "description": "This project is a web application thats displays the name and score of players using fetch data from API.",
+  "scripts": {
+    "build": "webpack --mode development",
+    "start": "webpack serve --open"
```

- [ ] These scripts sections should be something similar to the following
```suggestion
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
```

---

Alright thanks for the reviews